CMakeLists.txt

```diff
@@ -204,6 +204,11 @@ else()
             -Wno-maybe-uninitialized \
             ")
         endif()
+        if(CMAKE_CXX_COMPILER_ID MATCHES "Clang")
```

Comment:
Should this be its own PR?

Reply:
Never mind, I see it was added in this [commit](https://github.com/Point72/csp/commit/eaed6683dd4b4cd22c05a5cb9df0a748f416783f).